products/sle/main.pm

```diff
@@ -698,6 +698,17 @@ sub load_consoletests() {
     }
 }
 
+# possibility to run as part of aggregated tests
+sub run_aggregated_maintenance_tests() {
+    use List::Util qw(all);
+    my %test_repos = get_env_by_pattern("\\w+_TEST_REPO");
```

Comment:
why we have this function in utils ? isn't utils is a storage for something WIDELY used in project ?